org.envirocar.app/src/org/envirocar/app/views/recordingscreen/RecordingScreenActivity.java

```diff
@@ -148,7 +148,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         // init state
         this.bluetoothImage.setSelected(false);
-        this.gpsImage.setEnabled(false);
+        this.gpsImage.setEnabled(true);
```

Comment:
Well, GPS icon indicates whether there is a GPS connection or not. It does not indicate, whether GPS is enabled or disabled. However, the state will be changed here: https://github.com/enviroCar/enviroCar-app/blob/98403d7bc73e246bdae761e515c78c3e688a0dfa/org.envirocar.app/src/org/envirocar/app/views/recordingscreen/RecordingScreenActivity.java#L227-L235

So please, revert your change, since it don't have any effect.

Reply:
oh, I didn't know that thanks for letting me know :) 
I will surely revert the change